pages/index.js

```diff
@@ -16,26 +17,27 @@ export default function Home({ recipes }) {
 
         {/* show all recipes */}
         <div className="recipe-card">
-          {recipes.map((recipe, index) => (
-            <Link href={`/recipes/${recipe.slug.current}`} key={index}>
-              <a>{recipe.name}</a>
+          {recipes.map(recipe => (
+            <Link href={`/recipes/${recipe?.slug?.current}`} key={recipe?._id}>
```

Comment:
Added optional chaining to guard against accessing attributes on undefined variables. I also replaced the index key with a unique one using the recipe `_id`. This is especially important when we have live preview since you can have reordering of these list items. https://reactjs.org/docs/reconciliation.html#keys

Reply:
Makes sense